src/views/v2/month/calendar-body/day/calendar-events/calendar-event.php

```diff
@@ -16,7 +16,7 @@
  * @see tribe_get_event() For the format of the event object.
  */
 
-$classes = get_post_class( [ 'tribe-events-calendar-month__calendar-event' ], $event->ID );
+$classes = tribe_get_post_class( [ 'tribe-events-calendar-month__calendar-event' ], $event->ID );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 974a393e0bd5bbb2e57aedbb21e1f9f3</sub>

Reply:
Ignoring.